### PR TITLE
add link to rfc-0001 in broker README

### PR DESCRIPTION
**Pull request by takahser**
# Description
- What does this PR do?
  - link added
- Why are these changes needed?
  - improve docs
- How were these changes implemented and what do they affect?
  - only concerns docs

### CODE REVIEW

**Review by ggwpez — APPROVED** [review]
We should also invest in a proper pallet documentation, currently there is none 🙈

**Comment by takahser** [comment]
@ggwpez here you go.

File: substrate/frame/broker/README.md [hunk 1]
```diff
@@ -1,15 +1,102 @@
...
+- `partition` - Split owned Coretime.
+- `interlace` - Interlace owned Coretime into alternating slots.  
+- `assign` - Assign owned Coretime to a Task.
+- `pool` - Contribute owned Coretime to Instantaneous pool.
+- `claim_revenue` - Claim revenue from Instantaneous pool contribution.
+- `purchase_credit` - Purchase Instantaneous Coretime credit.
+
+## Events
+
+You're absolutely right, I missed covering some of the events in the `Event` enum in the Rust code. Let me add the missing events:
```
Comment (joepetrowski) [inline substrate/frame/broker/README.md:49]: ?

**Review by ggwpez — CHANGES_REQUESTED** [review]
Uhm with "pallet documentation" i meant this: https://github.com/paritytech/polkadot-sdk/blob/master/docs/DOCUMENTATION_GUIDELINES.md#pallet-crates  
README files are obsolete unter this schema. So either rollback or go all the way please.

File: substrate/frame/broker/README.md (hunk 1 above)
Comment (takahser) [inline substrate/frame/broker/README.md:49]:
```suggestion
```

**Comment by takahser** [comment]
@ggwpez ok thx, updated the following:

- rolled back changes
- fixed another broken link

File: substrate/frame/broker/README.md (hunk 1 above)
Comment (takahser) [reply substrate/frame/broker/README.md]: Thanks, LLM leftovers 😄.

**Review by ggwpez — APPROVED** [review]

**Review by joepetrowski — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Pull request merged**